findme-users/handler.py

```diff
@@ -53,6 +60,55 @@ def get_individual_user():
     return user_service.get_user(user_id)
 
 
+@app.get("/users/search")
+@tracer.capture_method
+@authorizer.requires_auth(app=app)
+def get_similar_users():
+    username = app.current_event.query_string_parameters.get("username")
+    return user_service.get_similar_users(username)
+
+
+@app.put("/users/<user_id>/follow")
+@tracer.capture_method
+@authorizer.requires_auth(app=app)
+def follow_user(user_id: Annotated[int, Path(lt=999)]):
+    requestee_id = app.context.get('claims').get('sub')
```

Comment:
if pipe is in userId -> split according to pipe -> see examples on dev branch

Reply:
Or  like in function below

Reply:
Fixed in [a1c5ac2](https://github.com/uzh-ase-fs24/backend/commit/a1c5ac223339801e97dc498127287601e7df2c92)